pkg/scheduler/apis/config/v1alpha1/defaults.go

```diff
@@ -103,4 +103,12 @@ func SetDefaults_KubeSchedulerConfiguration(obj *kubeschedulerconfigv1alpha1.Kub
 		defaultBindTimeoutSeconds := int64(600)
 		obj.BindTimeoutSeconds = &defaultBindTimeoutSeconds
 	}
+
+	if obj.PodInitialBackoffDurationSeconds == 0 {
+		obj.PodInitialBackoffDurationSeconds = kubeschedulerconfigv1alpha1.SchedulerDefaultPodInitialBackoffDurationSeconds
```

Comment:
why not the values from "k8s.io/kube-scheduler/config/v1alpha1"?

Reply:
I think it is the value from  "k8s.io/kube-scheduler/config/v1alpha1"